crates/burn-core/src/nn/sigmoid.rs

```diff
@@ -0,0 +1,39 @@
+use crate as burn;
+
+use crate::module::Module;
+use crate::tensor::backend::Backend;
+use crate::tensor::Tensor;
+
+/// Applies the rectified linear unit function element-wise
+/// See also [sigmoid](burn::tensor::activation::sigmoid)
+///
```

Comment:
Should state: "Applies the logistic sigmoid function element-wise"

Also, we can remove the empty comment line `///`